custom_model_runner/datarobot_drum/drum/common.py

```diff
@@ -73,10 +73,30 @@ class UnstructuredDtoKeys:
     CHARSET = "charset"
 
 
+class StructuredDtoKeys:
+    FILENAME = "filename"
+    BINARY_DATA = "binary_data"
+    MIMETYPE = "mimetype"
+    CHARSET = "charset"
+
+
 class PredictionServerMimetypes:
     APPLICATION_JSON = "application/json"
     APPLICATION_OCTET_STREAM = "application/octet-stream"
     TEXT_PLAIN = "text/plain"
+    TEXT_ARROW = "text/arrow"
+    TEXT_MTX = "text/mtx"
```

Comment:
aren't these binary formats? would application/[mtx, arrow] make more sense? text is intended for human readable formats

Reply:
Agree.
We talked with @rvorobii and @klichukb and decided that as long as there is not official mimetypes for these formats we'll stick with something.
Let's wait for others to comment and fix it.

Reply:
no need. if it's been discussed and agreed to, i don't feel strongly

Reply:
this is definitely not "text".

I propose to change it to "application/x-apache-arrow-stream".
- "application" because it's a binary format
- "x-" means it's not standardized (check https://en.wikipedia.org/wiki/Media_type)
- "-stream" because it's a stream in Arrow terms

check out this thread where people are discussing mime types for Arrow https://lists.apache.org/thread.html/b15726d0c0da2223ba1b45a226ef86263f688b20532a30535cd5e267%40%3Cdev.arrow.apache.org%3E

Reply:
This is _not_ a text format. When we discussed sticking to some I mean one of those discussed in arrow threads. ^ What Roman says.

Reply:
Also mtx is probably not text as well, these ought to be fixed before merging.

Reply:
Mtx seems to be a text format, maybe that one is fine with 'text/' unless you know better.